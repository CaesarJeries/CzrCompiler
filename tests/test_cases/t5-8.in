void main()
{
    // switch test

     int x = 8;
     switch(x)
     {
        case 2 : print("a") ;  break;
        case 8b: {print("b"); print("\nok"); }   break;
        case 8: print("c");   break;
        case 100b: print("d"); break;
        case 300: print("e"); break;
     };
     print("\n");

     x = 0 - 10;
     byte y = 11 b;
     switch(x+y)
     {
            case 0 : print("a") ;  break;
            case 1b:
            switch(50b)
            {
                case 50: print("b") ; break;
            };
            break;
            case 30: print("c") ;   break;
            case 10 b: print("d") ; break;
     };
     print("\n");


}