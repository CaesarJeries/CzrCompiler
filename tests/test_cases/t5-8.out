Loaded: ./exceptions.s
b
ok
b
